db/migrations/20220817104047_user_address.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE user_address (
```

Comment:
```suggestion
CREATE TABLE user_addresses (
```
Table명은 항상 복수로 작성해 주세요~!